CMake/FindPackages.cmake

```diff
@@ -79,9 +79,6 @@ if(Boost_name)
   if(NOT "${${Boost_name}_INCLUDE_DIRS}" MATCHES "-NOTFOUND")
     include_directories(SYSTEM ${${Boost_name}_INCLUDE_DIRS})
   endif()
-  if(NOT "${${Boost_name}_INCLUDE_DIR}" MATCHES "-NOTFOUND")
```

Comment:
The only difference is that DIR became DIRS. I think DIRS is the new one and it is (to me) more right since DIRS is a overset of DIR.